src/test/kotlin/org/rust/lang/core/dfa/RsControlFlowGraphTest.kt

```diff
@@ -1098,6 +1098,6 @@ class RsControlFlowGraphTest : RsTestBase() {
         val cfg = ControlFlowGraph.buildFor(function.block!!, getRegionScopeTree(function))
         val expected = expectedIndented.trimIndent()
         val actual = cfg.graph.depthFirstTraversalTrace(cfg.entry)
-        check(actual == expected) { throw ComparisonFailure("Comparision failed", expected, actual) }
+        check(actual == expected) { throw ComparisonFailure("Comparison failed", expected, actual) }
```

Comment:
BTW why we don't use `assertEquals` here?

Reply:
I'm not sure, probably I've just copied this code from another test.
Similar code appears in `RsMacroGraphWalkerTest` and `RsMacroGraphBuilderTest` as well, so I can open another PR to change them all.

Reply:
Done: https://github.com/intellij-rust/intellij-rust/pull/7027